Microsoft Defender for Endpoint deprecation (#37727)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

---
Use the Microsoft Defender for Endpoint (previously Microsoft Defender Advanced Threat Protection (ATP)) integration for preventative protection, post-breach detection, automated investigation, and response.

## Deprecation Announcement

Note: Following [this](https://learn.microsoft.com/en-us/defender-endpoint/configure-siem) announcement by Microsoft about migrating from the deprecated SIEM API to the Graph API, we are deprecating the following:

- **14 commands**
- **Fetch-incidents functionality**

### Replacement Options:
- Some commands have direct replacements in the **Microsoft Graph Security** integration.
- Others do not have exact replacements but offer alternatives that return similar data.
- A few commands have no available replacements.

> Below, you'll find details on the deprecated commands and their replacements or alternatives.
> [Go to details](#deprecation-details)

Reviewer: ```suggestion
[See Deprecation Details](#deprecation-details) to find details on the deprecated commands and their replacements or alternatives.

```

- comments
- evidence

## Deprecation Announcement

Note: Following [this](https://learn.microsoft.com/en-us/defender-endpoint/configure-siem) announcement by Microsoft about migrating from the deprecated (and possibly end-of-life) SIEM API to the Graph API, we are deprecating the following:

- **14 commands**
- **Fetch-incidents functionality**

### Replacement Options:
- Some commands have direct replacements in the **Microsoft Graph Security** integration.
- Others do not have exact replacements but offer alternatives that return similar data.

> - A few commands have no available replacements:

Reviewer: ```suggestion
- A few commands have no available replacements.
```

## Deprecation Details Table

| **Command**                     | **Replacement**                                                                                                                                               |
|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `Fetch incidents`    | Use the `Fetch incidents` in the `Microsoft Graph Security` integration instead.                                                                                                                                    |

Reviewer: ```suggestion
| `Fetch incidents`    | Use `Fetch incidents` in the `Microsoft Graph Security` integration instead.                                                                                                                                    |
```
